README.md

```diff
@@ -1 +1,21 @@
-# MJC
+# MJC-3-multimodule-spring-boot-hibernate-jpa-hateoas-pagination
+
+There is a multimodule application which include following modules:
+
+- DAO - default port 8092;
+- SERVICE - default port 8093;
```

Comment:
No applications needed in DAO and SEVICE. It is enough to have a UI in WEB for the whole app